gmaps/directions.py

```diff
@@ -88,17 +101,22 @@ def _handle_layer_status(self, change):
                 "No directions returned: " + change["new"])
 
 
-def _directions_options(start, end, waypoints):
+def _directions_options(start, end, waypoints, opts=None):
     start = tuple(start)
     end = tuple(end)
     if waypoints is None:
         data = [start, end]
     else:
         data = [start] + locations_to_list(waypoints) + [end]
-    return {"data": data}
+    
+    model = {"data": data}
+    
+    if opts is not None:
+        model.update(opts)
+    return model
 
 
-def directions_layer(start, end, waypoints=None):
+def directions_layer(start, end, waypoints=None, **kwargs):
```

Comment:
I'm in favour of explicitly specifying the kwargs (ie explicitly writing `avoid_tolls=False, avoid_highways=False, ...`. 

It's a bit easier to understand for someone reading the code, and since Jupyter and many IDEs offer completion based on the signature, having the kwargs spelled out makes them much more discoverable.